pkg/users/delete-group-dialog.js

```diff
@@ -49,12 +49,16 @@ export function delete_group_dialog(group) {
                 caption: group.userlistPrimary.length > 0 ? _("Force delete") : _("Delete"),
                 style: "danger",
                 clicked: () => {
-                    const prog = ["/usr/sbin/groupdel"];
+                    const prog = ["groupdel"];
                     if (group.userlistPrimary.length > 0)
                         prog.push("-f");
                     prog.push(group.name);
 
-                    return cockpit.spawn(prog, { superuser: "require", err: "message" });
+                    return cockpit.spawn(prog, { superuser: "require", err: "message" })
+                            .then(() => {
+                                cockpit.location.go("/");
+                                return Promise.resolve();
+                            });
```

Comment:
This is an independent bug fix, right? Deleting a group previously ended up at a broken page? I think that corresponds to the "Deleting group will go back to overview" part of the integration test. Would be nice to split out into a separate commit (or even PR, if you want to spoon-feed this some more)

OOI, why does this have to return a resolved promise, as opposed to just `return true`?

Reply:
These are equivalent, indeed 'return true' is cleaner. Replaced.

Reply:
It did not end up in broken page, because before we were not in the details, so no page redirection was needed. This is now needed, because the 'delete group dialog' can now also have the group details page as entry.

Reply:
I see, thanks! So let's keep it in here indeed.